usagov/usagov-2021
# USAGOV-1821: Callout Icon Resize

## Jira Task

[USAGOV-1821](https://cm-jira.usa.gov/browse/USAGOV-1821)

## Description


Updating size of the callout icon on the government benefits page

## Type of Changes

 
<img width="1480" alt="Screenshot 2024-08-13 at 3 55 21 PM" src="https://github.com/user-attachments/assets/6c81ba41-5dda-45d7-9426-e2f7de69cfe1">


## Testing Instructions



1. start up local and rebuild the Drupal theme

### Change Requirements



### Validation Steps

- visit `benefits` and inspect that the icon is now set to 45x45px
- visit home page and inspect the same icon and verify that it was not changed from 54px
 
## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
